Packages/vcs/Lib/vtk_ui/manager.py

```diff
@@ -14,7 +14,15 @@ def __init__(self, interactor):
         self.renderer = vtk.vtkRenderer()
         self.renderer.SetViewport(0, 0, 1, 1)
         self.renderer.SetBackground(1, 1, 1)
+
+        # Used to overlay actors above widgets
+        self.actor_renderer = vtk.vtkRenderer()
+        self.actor_renderer.SetViewport(0, 0, 1, 1)
+        self.actor_renderer.SetBackground(1, 1, 1)
```

Comment:
I added a second renderer to the UI Manager because (between the switchover from vtkTextWidget/Representation to vtkTextActor and the Line object) I was using some straight actors in my UI code, and they were showing up below the Widgets.